### PR TITLE
feat: add color to github actions output

### PR DESCRIPTION
**Pull request by kevin-s31**
Added the FORCE_COLOR environment variable and set it to true to see a colored output for a snyk test in github actions

### CODE REVIEW

File: setup/setup_snyk.sh [hunk 1]
```diff
@@ -49,6 +49,7 @@ esac
     echo "#!/bin/bash"
     echo export SNYK_INTEGRATION_NAME="GITHUB_ACTIONS"
     echo export SNYK_INTEGRATION_VERSION=\"setup \(${2}\)\"
+    echo export FORCE_COLOR= True
```
Comment (garethr) [inline setup/setup_snyk.sh:52]: I think the space after `=` will cause this to not work?

**Review by garethr — CHANGES_REQUESTED** [review]

**Comment by kevin-s31** [comment]
Will check on both of the above.

**Comment by kevin-s31** [comment]
Removed the space after the variable and also instead of True, set it to 2 to allow for 256 color ANSI output

**Review by bastiandoetsch — APPROVED** [review]
LGTM

**Pull request merged**